### DISCUSSION TITLE
P2 (attaque) ouverte !

### DISCUSSION THREAD

**Discussion by beng-git** [danymat/INSAnonym-utils#102] [General]
Bonsoir,

La phase d'attaque est ouverte : vous pouvez télécharger et soumettre des attaques sur les soumissions des autres équipes. Le tableau des scores est mis à jour en direct.  Le traitement d'une attaque est très rapide donc les résultats arrivent rapidement.

Si vous avez des problèmes de format de fichier d'attaque, vous pouvez l'indiquer sur ce forum.

Cordialement,

L'équipe DARC

**Comment by beng-git** [comment]
Outre les équipes n'ayant pas soumis, les équipes hors compétition (compétition précédente APVP 2021) sont : 

- PrivUQAM
- InsaLyon1
- Femtorange
- BVT

Vous pouvez bien sûr faire des attaques contre ces équipes si vous le souhaitez, toutefois nous ne prendrons pas en compte ce score dans les résultats.

**Comment by SIanas1964** [comment]
Monsieur je n'arrive pas à télécharger les bases de données des autres équipes !!
Par contre je ne trouve pas de probléme pour télécharger les bases de données de la compétition précédente.

**Comment by ceichler** [comment]
Bonjour,

Je vous confirme ne pas avoir de problème pour télécharger les bases de données soumises par les autres compétiteurs, que cela soit en tant qu'admin ou simple compétiteur. Si vous souhaitez de l'aide pour troubleshooter votre problème je vous invite à nous communiquer plus de précision.

Cdlt

**Comment by beng-git** [comment]
Bonjour,

Après quelques tests : j'ai réussi à télécharger avec Firefox, mais lorsque j'essaie de télécharger avec Edge (en cliquant sur le bouton "source") j'ai un message d'erreur "couldn't download, network issue". Je vous invite donc à essayer de télécharger avec un autre navigateur (par exemple Firefox). Par contre je peux soit tout télécharger, soit rien.

J'ai aussi réussi à télécharger avec edge en faisant clic droit sur le bouton "source" et en choisissant "ouvrir dans un autre onglet".

Cordialement,

BN

**Comment by beng-git** [comment]
Pour plus de clarté voici (sauf erreur) la liste des équipes en compétition : 

- 404
- anonym212
- autofill
- DAZC
- KodaTeam
- MichelLardon
- OSS
- pocochocambo
- ServiceInactifAnonyme
- Sibyl
- THAThack
- THE
- TresAnonyme
- VietRoc

**Comment by danymat** [comment]
> Vous pouvez bien sûr faire des attaques contre ces équipes si vous le souhaitez, toutefois nous ne prendrons pas en compte ce score dans les résultats.
 
@beng-git Pour information, cela ne marchera pas pour des problèmes de rollback. En effet, la compétition précédente utilisait des tableaux de comparaison (pour le F file) plutôt qu'un json.

**Comment by beng-git** [comment]
Bonsoir,

dû à un problème réseau à l'INSA la compétition n'a pas pu être clôturée à temps, toutefois elle le sera dès que possible.

Merci à tous les participants !
